index.jsx

```diff
@@ -22,10 +23,15 @@ class Variable extends React.Component {
     this.props.changeSelected(event.target.value);
   }
 
+  getName() {
+    return this.props.name || this.props.variable;
```

Comment:
does this need a third fallback in case `name` and `variable` are both missing?

Reply:
maybe itll be okay because doing `arr[undefined]` just returns `undefined`.

Reply:
It ought to be a runtime or compile time error? Should I throw like a Props missing error?

Reply:
ya that might be a good idea